java/connecting-to-bigtable/src/main/resources/hbase-site.xml

```diff
@@ -0,0 +1,25 @@
+<!--[START bigtable_hbase_site_xml]-->
+<configuration>
+  <property>
+    <name>hbase.client.connection.impl</name>
+    <value>com.google.cloud.bigtable.hbase1_x.BigtableConnection</value>
+  </property>
+  <property>
+    <name>google.bigtable.project.id</name>
+    <value>${bigtable.projectID}</value>
```

Comment:
its worth noting that `bigtable.projectID`, `bigtable.instanceID` and `bigtable.appProfileID` can be passed in as system properties. else seems unclear where these come from.

Reply:
Done